man/ipsec.conf.5.in

```diff
@@ -471,6 +471,16 @@ Note that fragmented IKE messages sent by a peer are always accepted
 irrespective of the value of this option (even when set to
 .BR no ).
 .TP
+.BR if_id_in " = <id>"
```

Comment:
The whole ipsec.conf support for if_id is probably not going to be merged.

Reply:
Ok